### DISCUSSION TITLE
No SMTP servers were available - No route to host

### DISCUSSION THREAD

**Discussion by nvdg** [postalserver/postal#3077] [Help with using Postal]
All my emails including test emails to mail-tester.com keep giving me the soft fail with message:

No SMTP servers were available for srv1.mail-tester.com. Tried reception.mail-tester.com.
```
No route to host - connect(2) for "94.23.206.89" port 25
Support Ref: EYLEW1VO
```

I've checked other messages here and see mostly port 25 being blocked by the service provider, but I'm hosting it on a server at home and from what I can see nothing is blocking any outbound ports.
All DNS checks turn up correct and green.

Any other leads to what to check?

**Comment by nvdg** [comment]
Alright, can close it myself again... I thought no one would block it when self hosting, turns out my and most other internet providers also block it...

**Reply by willpower232** [reply]
Yes unfortunately you need either an email friendly server provider or an email friendly internet provider :disappointed:
